tests/sub_port_interfaces/templates/tunnel_config.j2

```diff
@@ -0,0 +1,13 @@
+{
+    "TUNNEL": {
+{% for ip in tunnel_addr_list %}
+        "{{ "MuxTunnel" + loop.index|string }}": {
+            "dst_ip" : "{{ ip }}",
+            "ttl_mode": "pipe",
+            "ecn_mode": "copy_from_outer",
+            "dscp_mode": "pipe",
+            "tunnel_type": "IPINIP"
+        }{% if not loop.last %},{% endif %}
+{% endfor %}
```

Comment:
An empty line should be needed between  10 and  11. Can you check `sonic-cfggen -t` output?

Reply:
Sure. Output:
```
$ sonic-cfggen -t tunnel_config.j2 
{
    "TUNNEL": {

        "MuxTunnel1": {
            "dst_ip" : "172.16.0.9",
            "ttl_mode": "pipe",
            "ecn_mode": "copy_from_outer",
            "dscp_mode": "pipe",
            "tunnel_type": "IPINIP"
        },

        "MuxTunnel2": {
            "dst_ip" : "172.16.4.13",
            "ttl_mode": "pipe",
            "ecn_mode": "copy_from_outer",
            "dscp_mode": "pipe",
            "tunnel_type": "IPINIP"
        }

    }
}
```
Do we need to add an empty line between 10 and 11?

Reply:
Tried latest `sonic-cfggen` in vs docker. Vs docker built @sonic-buildimage master 1863e1fa2
```
# which sonic-cfggen 
/usr/local/bin/sonic-cfggen
# cat /usr/local/bin/sonic-cfggen       
#!/usr/bin/python3
```

#### Without empty line
```
$ cat tunnel_config.j2 
{
    "TUNNEL": {
{% for ip in tunnel_addr_list %}
        "{{ "MuxTunnel" + loop.index|string }}": {
            "dst_ip" : "{{ ip }}",
            "ttl_mode": "pipe",
            "ecn_mode": "copy_from_outer",
            "dscp_mode": "pipe",
            "tunnel_type": "IPINIP"
        }{% if not loop.last %},{% endif %}
{% endfor %}
    }
}
```
output
```
sonic-cfggen -t tunnel_config.j2 -a "{\"tunnel_addr_list\": [\"172.16.0.9\", \"172.16.4.13\"]}"
{
    "TUNNEL": {
        "MuxTunnel1": {
            "dst_ip" : "172.16.0.9",
            "ttl_mode": "pipe",
            "ecn_mode": "copy_from_outer",
            "dscp_mode": "pipe",
            "tunnel_type": "IPINIP"
        },        "MuxTunnel2": {
            "dst_ip" : "172.16.4.13",
            "ttl_mode": "pipe",
            "ecn_mode": "copy_from_outer",
            "dscp_mode": "pipe",
            "tunnel_type": "IPINIP"
        }    }
}
```

#### With empty line
```
$ cat tunnel_config.j2 
{
    "TUNNEL": {
{% for ip in tunnel_addr_list %}
        "{{ "MuxTunnel" + loop.index|string }}": {
            "dst_ip" : "{{ ip }}",
            "ttl_mode": "pipe",
            "ecn_mode": "copy_from_outer",
            "dscp_mode": "pipe",
            "tunnel_type": "IPINIP"
        }{% if not loop.last %},{% endif %}

{% endfor %}
    }
}
```
Pretty output
```
sonic-cfggen -t tunnel_config.j2 -a "{\"tunnel_addr_list\": [\"172.16.0.9\", \"172.16.4.13\"]}"
{
    "TUNNEL": {
        "MuxTunnel1": {
            "dst_ip" : "172.16.0.9",
            "ttl_mode": "pipe",
            "ecn_mode": "copy_from_outer",
            "dscp_mode": "pipe",
            "tunnel_type": "IPINIP"
        },
        "MuxTunnel2": {
            "dst_ip" : "172.16.4.13",
            "ttl_mode": "pipe",
            "ecn_mode": "copy_from_outer",
            "dscp_mode": "pipe",
            "tunnel_type": "IPINIP"
        }
    }
}
```

You can ignore if you do not see the above difference.

Reply:
It makes sense. Added an empty line to tunnel_config. Thanks for your help in investigating.